src/components/Editor/CallSites.js

```diff
@@ -58,22 +57,18 @@ class CallSites extends Component {
     const { editor } = this.props;
     const codeMirrorWrapper = editor.codeMirror.getWrapperElement();
 
-    if (features.columnBreakpoints) {
-      codeMirrorWrapper.addEventListener("click", e => this.onTokenClick(e));
-      document.body.addEventListener("keydown", this.onKeyDown);
-      document.body.addEventListener("keyup", this.onKeyUp);
-    }
+    codeMirrorWrapper.addEventListener("click", e => this.onTokenClick(e));
+    document.body.addEventListener("keydown", this.onKeyDown);
+    document.body.addEventListener("keyup", this.onKeyUp);
   }
 
   componentDidUnMount() {
```

Comment:
hm, correct me if I'm wrong here, but there is no `componentDidUnMount` lifecycle method in React. Only [componentWillUnmount](https://reactjs.org/docs/react-component.html#componentwillunmount).

Reply:
Nice catch! Thanks!